### ISSUE TITLE
adjust_gamma will change 1x1 array to a float

### ISSUE THREAD

**Issue by jinyu121** [scikit-image/scikit-image#4166]
## Description
In some cases, `adjust_gamma` will change a 1x1 array to a float number.

## Way to reproduce
```python
img = np.ones([1,1]) # Notice: no dtype provided here
print(img) # It is a 1x1 array
img = adjust_gamma(img, 1.5)
print(img) # It is a float number
```

And if you provide the dtype, all is well.

## Version information
```python
3.7.4 (default, Aug 13 2019, 15:17:50) 
[Clang 4.0.1 (tags/RELEASE_401/final)]
Darwin-18.7.0-x86_64-i386-64bit
scikit-image version: 0.15.0
numpy version: 1.16.4
```

**Comment by sciunto** [comment]
@jinyu121 Thanks for reporting. What do you mean by "in some cases"? Is there cases where a 1x1 array is not turned into a float?

**Comment by sciunto** [comment]
@jinyu121 I proposed a patch.

**Comment by jinyu121** [comment]
Case 1:
```python
img = np.ones([1,1]) # Notice: no dtype provided here
print(img) # It is a 1x1 array
img = adjust_gamma(img, 1.5)
print(img) # It is a float number
```

Case 2:
```python
img = np.ones([1,1], dtype=np.float32) # Notice:provide the dtype here
print(img) # It is a 1x1 array
img = adjust_gamma(img, 1.5)
print(img) # It is still a 1x1 array
```

**Comment by jni** [comment]
@jinyu121 thanks for the report! @sciunto has now fixed this in the development version in #4169, and this fix will be in the next release, which should happen in the next couple of weeks.
